jlm/hls/ir/hls.hpp

```diff
@@ -1019,7 +1024,7 @@ class mem_resp_op final : public jlm::rvsdg::simple_op
   virtual ~mem_resp_op()
   {}
 
-  mem_resp_op(const std::vector<const rvsdg::valuetype *> & output_types)
+  mem_resp_op(const std::vector<std::shared_ptr<const rvsdg::valuetype>> & output_types)
```

Comment:
`explicit`

Reply:
You resolved the conversation, but the explicit is not there.

Reply:
sorry, forgotten cherry-pick on branch, resolved now